storage-proofs-porep/Cargo.toml

```diff
@@ -41,6 +40,11 @@ libc = "0.2"
 fdlimit = "0.2.0"
 fr32 = { path = "../fr32", default-features = false }
 
+[target."cfg(target_arch = \"aarch64\")".dependencies]
+sha2 = { git = "https://github.com/RustCrypto/hashes.git", rev = "78f00c5f6d218", features = ["compress", "asm"] }
```

Comment:
It looks like the recent 0.9.3 release contains that commit, so you can now point to that version instead of a Git commit.

Reply:
Thank you for the review.

I guess we don't need `-Zfeatures=itarget` either, since the `sha2-0.9.3` has updated to use the instrinsic backend even if `asm` is enbled [2].
Use sha2-0.9.3 with feature ["compress", "asm"] would be enough, I will withdraw [1] if this PR can be integerated.

Regards.

[1] https://github.com/filecoin-project/filecoin-ffi/pull/163
[2] https://github.com/RustCrypto/hashes/pull/224